seed/serializers/filter_groups.py

```diff
@@ -0,0 +1,30 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2022, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.
+:author
+"""
+from rest_framework import serializers
+
+from seed.models import VIEW_LIST_INVENTORY_TYPE, FilterGroup
+from seed.models.filter_group import LABEL_LOGIC_TYPE
+
+
+class FilterGroupSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = FilterGroup
+        fields = ('name', 'query_dict', 'inventory_type', 'id', "organization_id", "labels", "label_logic")
+        extra_kwargs = {
+            'user': {'read_only': True},
+            'organization': {'read_only': True},
+            'organization_id': {'read_only': True}
+        }
+
+    def to_representation(self, instance):
+        ret = super().to_representation(instance)
```

Comment:
does `ret` stand for `return value`?

Reply:
yep, I took a cue from the `LabelSerialzer`, but looking at the code base it's not used anywhere else. I will change it to something more intuitive.